sdk/helper/keysutil/lock_manager.go

```diff
@@ -495,7 +499,7 @@ func (lm *LockManager) ImportPolicy(ctx context.Context, req PolicyRequest, key
 		}
 	}
 
-	err = p.Import(ctx, req.Storage, key, rand)
+	err = p.Import(ctx, req.Storage, key, req.IsCiphertextSet, rand)
```

Comment:
I'm not sure where to place it, so I'm just marking this comment here:

The issue with this particular approach is that `sdk/` is an externally-accessible and callable library that we provide and support. Changing `p.Import(...)` affects externally callable functions, and adding this boolean would break existing callers, if any.

I'd suggest not adding the boolean, but add another method instead.